src/app/(mobile-ui)/withdraw/manteca/page.tsx

```
@@ -136,10 +136,17 @@ export default function MantecaWithdrawFlow() {
         let isValid = false
         switch (countryPath) {
             case 'argentina':
-                const { valid, message } = validateCbuCvuAlias(value)
-                isValid = valid
-                if (!valid) {
-                    setErrorMessage(message!)
+                const argResult = validateCbuCvuAlias(value)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)